htdocs/compta/dons/class/don.class.php

```diff
@@ -701,13 +701,29 @@ function sum_donations($param)
 
 
     /**
-     *	Return clicable name (with picto eventually)
+     *  Return clicable name (with picto eventually)
      *
-     *	@param	int		$withpicto		0=Pas de picto, 1=Inclut le picto dans le lien, 2=Picto seul
-     *	@return	string					Chaine avec URL
+     *  @param  int     $withpicto      0=Pas de picto, 1=Inclut le picto dans le lien, 2=Picto seul
+     *  @return string                  Chaine avec URL
+     *  @deprecated
```

Comment:
Adding something like "Please use getHtmlLink() instead" to the line makes it more usefull in generated documentation where the code might not be next to the keyword.